lib/vm.js

```diff
@@ -32,7 +32,10 @@ const {
   ERR_INVALID_ARG_TYPE,
   ERR_VM_MODULE_NOT_MODULE,
 } = require('internal/errors').codes;
-const { isModuleNamespaceObject, isArrayBufferView } = require('util').types;
+const { 
+  isModuleNamespaceObject, 
+  isArrayBufferView 
```

Comment:
Try to check if there are spaces at the end of line, we should remove that as CI said.

![Screen Shot 2019-03-19 at 8 41 23 AM](https://user-images.githubusercontent.com/23313266/54572488-ce2e6000-4a22-11e9-8ce1-918fbfe4a974.png)